core/mondoo-azure-security.mql.yaml

```diff
@@ -224,54 +229,49 @@ queries:
       - uid: mondoo-azure-security-ssh-access-restricted-from-internet-api
     docs:
       desc: |
-        This check ensures that SSH on port 22 is not configured to allow access from anywhere with the CIDR block "0.0.0.0".
+        Restricting SSH access from the internet minimizes the risk of brute force and other unauthorized access attempts on Azure resources such as virtual machines. Public internet exposure of SSH, especially on the default port 22, significantly increases the attack surface. It's crucial to evaluate and restrict SSH access by configuring network security groups (NSGs) to allow only necessary connections, preferably from known, secure sources. Implementing stringent access controls for SSH enhances security by preventing attackers from using compromised VMs as a pivot point for broader network intrusions.
       audit: |
-        __cnspec run__
+       **Manual Audit via Azure Portal:**     
 
-        To audit Microsoft Azure with `cnspec run`:
+        **Checking via Network Security Groups (NSGs) for All Resources:**
+        1. From the main Azure Portal menu, navigate to **Network security groups**.
+        2. Select an NSG associated with the resources you want to check.
+        3. Go to the **Inbound security rules** section.
+        4. Review all the rules to ensure none allow SSH access (port 22, TCP) from broadly defined sources such as 'Any', '0.0.0.0', or 'Internet'.
+        5. Confirm that rules configured to permit SSH access are scoped to specific, known IP addresses or ranges to minimize exposure.
 
-        1. Run `az login` to authenticate with the correct Azure subscription and tenant.
-        2. Run this query:
+        **Checking Specific Virtual Machines:**
+        1. Log into the Azure Portal.
+        2. Navigate to **Virtual machines**.
+        3. Select a specific virtual machine to review.
+        4. Within the virtual machine's blade, click on **Networking** under the settings menu.
```

Comment:
@HRouhani Can we make sure we don't use "click" in these policies. This should be "select"

Reply:
OK sure, I will make sure we used select instead of click overall.